instrumentation/liberty/liberty-dispatcher/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/liberty/dispatcher/LibertyResponseWrapper.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.liberty.dispatcher;
+
+import com.ibm.wsspi.http.channel.values.StatusCodes;
+
+public class LibertyResponseWrapper {
```

Comment:
Why do you need a wrapper for a single value?

Reply:
To be consistent with request and connection that have wrappers. Additionally I only used liberty classes from the wrappers